pkg/proxy/ipvs/safe_ipset.go

```diff
@@ -1,104 +0,0 @@
-/*
-Copyright 2019 The Kubernetes Authors.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-package ipvs
-
-import (
-	"sync"
-
-	"k8s.io/kubernetes/pkg/proxy/ipvs/ipset"
-)
-
-type safeIpset struct {
```

Comment:
the problem is that there can be two different execs in parallel https://github.com/kubernetes/kubernetes/pull/80627/files#r318400672